CHANGELOG.develop

```diff
@@ -1483,9 +1483,8 @@ Other:
 - Fixed syntax typo in configuration (thanks to EMayej)
 - Added =org-babel= support (thanks to Michael Rohleder)
 - Remove broken ENSIME key bindings (thanks to Bjarke Vad Andersen)
-- Fixed yanking types (thanks to Bjarke Vad Andersen)
-- Fixed gtags related initialization (thanks to Guido Kraemer)
-- Added LSP Java backend (thanks to Ivan Yonchovski)
```

Comment:
Looks like you removed some changelog entries in the rebase, you need to put these back.